app/views/account/tasks/complete.js.erb

```diff
@@ -1,2 +1 @@
-$('#complete_tasks').append('<%= j render('/account/projects/show_tasks', task: @task, project: @project) %>');
-$('#incomplete_tasks #complete_task_<%= @task.id %>').remove();
+// $('#tasks').find('.task_<%= @task.id%>').html('<%= j render('/account/projects/show_tasks', task: @task, project: @project) %>');
```

Comment:
I think for same user partial must be refreshed instantly without websockets ... but maybe it's fast enough and works instantly

so maybe this JS will be removed